### DISCUSSION TITLE
Create HowTo for creating a community script properly

### DISCUSSION THREAD

**Discussion by IVRYSimon** [community-scripts/ProxmoxVE#72] [Ideas]
Hi guys, 
wouldnt it be a good idea to have a tutorial for creating new scripts for the community?
A quick step-by-step guide would be an awesome starter. 
What should you pay attention to for a successful commit, perhaps also a short example script etc.?

BR
Simon

**Comment by MickLesk** [comment]
Of cause. There will be a template next week. I can't say exactly when yet. Basically, this should make it easier to get started. However, many things are not applicable to a template.

**Reply by MickLesk** [reply]
no. Calm down
